from unittest import TestCase


class TestSchedyRetry(TestCase):
    def test_increment(self):
        self.fail()
